README.md

```diff
@@ -527,15 +530,9 @@ Add the following job in your `azure-pipelines.yaml` file
       # Run MegaLinter
       - script: |
           docker run -v $(System.DefaultWorkingDirectory):/tmp/lint \
-            -e GIT_AUTHORIZATION_BEARER=$(System.AccessToken) \
-            -e CI=true \
```

Comment:
**CI** variable is used within MegaLinter so should be kept in the docker run arguments:)

Reply:
@nvuillam hmm, interesting... I tried to figure out purpose of this env, and only found reference here: https://github.com/oxsecurity/megalinter/blob/main/mega-linter-runner/test/megalinter-cli.test.js#L43 what is not related with scanning.
I did all my tests on my Azure DevOps instance without it as well and not observed any issues :) maybe it's obsoleted?

Reply:
It is used here :)
![image](https://user-images.githubusercontent.com/17500430/229341190-7b4977ef-197c-4ec0-bd42-8df49c2b7acb.png)

Reply:
@nvuillam hah! I tried config.get("CI"), os.environ.CI, etc but not tried simple "CI" 😅 thanks

Regardless of that, I think CI is not necessary here and below code is enough:

```python
if config.get("CONSOLE_REPORTER_SECTIONS", "true") == "true":
```
because the rest part of the code check if tool is ci and not another rule and conditions apply
https://github.com/oxsecurity/megalinter/blob/main/megalinter/utils_reporter.py#L199
https://github.com/oxsecurity/megalinter/blob/main/megalinter/utils_reporter.py#L203
https://github.com/oxsecurity/megalinter/blob/main/megalinter/utils_reporter.py#L207

eventually to keep backward compatibility and get rid of CI env in pipeline's code:

```python
if is_ci() and config.get("CONSOLE_REPORTER_SECTIONS", "true") == "true":
```

```python
def is_ci() -> bool:
    return True if "CI" in os.environ or is_github_actions() or is_gitlab_ci() or is_azure_pipelines() else False
```

Reply:
Agreed :)
Can i let you do that in https://github.com/oxsecurity/megalinter/pull/2504 ? :)